### PR TITLE
add missing parenthesis in MOM_internal_tides, prelude to #33

### PR DESCRIPTION
**Pull request by raphaeldussin**
this PR fixes missing parenthesis in additions of more than 2 terms

### CODE REVIEW

File: src/parameterizations/lateral/MOM_internal_tides.F90 [hunk 1]
```diff
@@ -1056,7 +1056,7 @@ subroutine propagate(En, cn, freq, dt, G, US, CS, NAngle)
     speed(:,:) = 0.0
     do J=jsh-1,jeh ; do I=ish-1,ieh
       f2 = G%CoriolisBu(I,J)**2
-      speed(I,J) = 0.25*(cn(i,j) + cn(i+1,j) + cn(i+1,j+1) + cn(i,j+1)) * &
+      speed(I,J) = 0.25*((cn(i,j) + cn(i+1,j)) + (cn(i+1,j+1) + cn(i,j+1))) * &
```
Comment (Hallberg-NOAA) [inline src/parameterizations/lateral/MOM_internal_tides.F90:1059]: This set of parentheses is not rotationally invariant.  Instead, this should be `speed(I,J) = 0.25*((cn(i,j) + cn(i+1,j+1)) + (cn(i+1,j) + cn(i,j+1))) * &`, which is rotationally invariant.

File: src/parameterizations/lateral/MOM_internal_tides.F90 [hunk 2]
```diff
@@ -1272,110 +1272,111 @@ subroutine propagate_corner_spread(En, energized_wedge, NAngle, speed, dt, G, CS
...
+          aC = a1 + (a2 + (a3 + a4))
       endif
 
       ! energy weighting ----------------------------------------
-      a_total = aNE + aN + aNW + aW + aSW + aS + aSE + aE + aC
-      E_new(m) = (aNE*En(i+1,j+1) + aN*En(i,j+1) + aNW*En(i-1,j+1) + &
-                  aW*En(i-1,j) + aSW*En(i-1,j-1) + aS*En(i,j-1) + &
-                  aSE*En(i+1,j-1) + aE*En(i+1,j) + aC*En(i,j)) / (dx(i,j)*dy(i,j))
+      a_total = (((aNE + aN) + (aNW + aW)) + ((aSW + aS) + (aSE + aE))) + aC
+      E_new(m) = ((((aNE*En(i+1,j+1) + aN*En(i,j+1)) + &
```
Comment (Hallberg-NOAA) [inline src/parameterizations/lateral/MOM_internal_tides.F90:1376]: This set of parentheses is not rotationally invariant. If the expressions for the aNE, etc., were rotationally invariant, one rotationally invariant set of parentheses for a_total would be `a_total = (((aNE + aSW) + (aNW+aSE)) + ((aN + aS) + (aW +aE))) + aC)`.

File: src/parameterizations/lateral/MOM_internal_tides.F90 [hunk 3]
```diff
@@ -1272,110 +1272,111 @@ subroutine propagate_corner_spread(En, energized_wedge, NAngle, speed, dt, G, CS
...
           a4 = (yNW - yN)*(0.5*(xNW + xN))
-          aW = a1 + a2 + a3 + a4
+          aW = a1 + (a2 + (a3 + a4))
           ! southwest area
           a1 = (yCrn - yS)*(0.5*(xCrn + xS))
           a2 = (yS - ySW)*(0.5*(xS + xSW))
           a3 = (ySW - yW)*(0.5*(xSW + xW))
           a4 = (yW - yCrn)*(0.5*(xW + xCrn))
-          aSW = a1 + a2 + a3 + a4
+          aSW = a1 + (a2 + (a3 + a4))
```
Comment (Hallberg-NOAA) [inline src/parameterizations/lateral/MOM_internal_tides.F90:1281]: This expression for aSW is not rotationally invariant.  If I have done the algebra right, a mathematically equivalent but rotationally invariant form would be `aSW = 0.5 * ((yCrn - ySW) * (xS - xW) + (yW - yS) * (xCrn - xSW))`.

A similar comment would apply to all of the other areas.

**Comment by raphaeldussin** [comment]
will reopen clean PR

**Pull request closed without merging**